apps/automated-tests/tests/e2e/checkout/logIn.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { test } from "fixtures/fixtures";
+import { URLS, userEmail, userPassword } from "utils/constants";
+
+test.describe(
+  "LogIn",
```

Comment:
I'd call it like this:

```suggestion
  "Auth - Log in",
```